SIAB/io/read_input.py

```
@@ -155,7 +155,15 @@ def is_reference_systems_line(key, value):
                         if isinstance(value[2], list):
                             return True
         return False
-    
+
+    #               STRU1      dimer     8       2       1   1.8  2.0  2.3  2.8  3.8
+    refsys = r'^\s*(STRU\d+)\s+(\w+)\s+(\d+)\s+(\d+)\s+(\d+)\s+((\d+\.\d+\s+)+)$'
```

Ruff

Local variable `refsys` is assigned to but never used

Remove assignment to unused variable `refsys`

(F841)